### PR TITLE
Add filter by template to VM list view

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

Adding a filter to VM list view to filter according to the template

## 🎥 Demo

templates-filter

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: src/views/virtualmachines/utils/virtualMachineRowFilter.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
+import { TFunction } from 'i18next';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { OTHER } from '@kubevirt-utils/components/DiskModal/DiskFormFields/utils/constants';
```
Comment (yaacov) [inline src/views/virtualmachines/utils/virtualMachineRowFilter.ts:4]: not sure we want to use OTHER from disks :-)

Comment (avivtur) [reply src/views/virtualmachines/utils/virtualMachineRowFilter.ts]: changed to using const other = t('Other') instead

**Pull request merged**